include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -3108,23 +3156,36 @@ MGTwoLevelTransfer<dim, LinearAlgebra::distributed::Vector<Number>>::
       this->vec_fine.reinit(0);
       this->partitioner_fine = external_partitioner_fine;
     }
-  else if (is_partitioner_contained(this->partitioner_fine,
-                                    external_partitioner_fine))
+  else if (internal::is_partitioner_contained(this->partitioner_fine,
+                                              external_partitioner_fine))
     {
       this->vec_fine.reinit(0);
 
-      for (auto &i : level_dof_indices_fine)
+      for (auto &i : this->level_dof_indices_fine)
         i = external_partitioner_fine->global_to_local(
           this->partitioner_fine->local_to_global(i));
 
       this->partitioner_fine_embedded =
-        create_embedded_partitioner(this->partitioner_fine,
-                                    external_partitioner_fine);
+        internal::create_embedded_partitioner(this->partitioner_fine,
+                                              external_partitioner_fine);
 
       this->partitioner_fine = external_partitioner_fine;
     }
 }
 
```

Comment:
Can you please add two more empty lines here, so that we have three empty lines between different functions?